manifests/server/install.pp

```diff
@@ -31,6 +31,17 @@
     }
   }
 
+  # Prevent the master service running and preventing Apache from binding to the port
+  if $::puppet::server::passenger and $::osfamily == 'Debian' {
+    file { '/etc/default/puppetmaster':
```

Comment:
Yes, I'd agree except that I think it has to run before the package installation, or it needs reimplementing into debconf (if possible, I've not checked).

It used to be in puppet::config::passenger but had a `before => Class['puppet::server::install']` relationship, which caused a loop when I contained the puppet::config::passenger class. I had to move it here to better match the relationship it had.